site/content/blog/2020-06-04-svelte-and-typescript.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Svelte <3 TypeScript
+description: Typernetically enhanced web apps
+author: Orta Therox
+authorURL: https://twitter.com/orta
+---
+
+It's been on the TODO list for a while, and it's now happening. TypeScript with Svelte is now a first class citizen of the ecosystem.
+
+We think it'll help you handle much larger Svelte code bases regardless of whether you use TypeScript or JavaScript.
+
+<figure>
+	<img alt="Screenshot of TypeScript in Svelte" src="media/svelte-ts.png">
+	<figcaption>Image of TypeScript + Svelte in VS Code (theme is <a href="https://marketplace.visualstudio.com/items?itemName=karyfoundation.theme-karyfoundation-themes">Kary Pro</a>.)</figcaption>
+</figure>
+
```

Comment:
I think a lot of people will want to just try it out straight away, so here (above the fold) would be a good place to have the quickstart guide

```suggestion

## Try it now

You can start a new Svelte TypeScript project using the [normal template](https://github.com/sveltejs/template) by running `node scripts/setupTypeScript.js` before you do anything else:

    npx degit sveltejs/template svelte-typescript-app
    cd svelte-typescript-app
    node scripts/setupTypeScript.js
    
If you're a VS Code user, make sure you're using the (new) [official extension](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode), which replaces the popular extension by James Birtles.

Later in this blog post, we'll detail the individual steps involved in using TypeScript in an existing Svelte project.

```

(side-note: how do you put code blocks inside a commit suggestion...?)

Reply:
Possibly by giving the `suggestion` language 4 backticks instead of 3. But don't quote me on this.

Reply:
This week I learned how:

GFMarkdown allows for the code fences to be an arbitrary size

````
```ts

```
````